src/index.js

```diff
@@ -197,6 +197,49 @@ module.exports = {
     'react/prefer-es6-class': 'error',
     'react/react-in-jsx-scope': 'error',
     'react/self-closing-comp': 'error',
+    'react/sort-comp': [2 , {
+      order: [
+        'static-methods',
+        'initialization',
+        'everything-else',
+        '/^on.+$/',
```

Comment:
Event handle method names should start with `handle`, not with `on`, so should we have this on the rules?